core/iwasm/aot/aot_runtime.h

```diff
@@ -251,6 +251,10 @@ typedef struct AOTModule {
     HashMap *ref_type_set;
     struct WASMRttType **rtt_types;
     korp_mutex rtt_type_lock;
+
+    /* array init data info */
+    uint32 array_init_data_count;
+    AOTArrayInitData **array_init_data_list;
```

Comment:
No need to add new data list, it isn't loaded and destroyed?
Per my understanding, we should use the below fields in AOTModule:
```C
    /* init data */
    uint32 mem_init_data_count;
    AOTMemInitData **mem_init_data_list;
```